package.json

```diff
@@ -45,6 +45,7 @@
     "babel-core": "^5.1.10",
     "babel-eslint": "^3.0.1",
     "babel-loader": "^5.0.0",
+    "babel-plugin-object-assign": "^1.1.0",
```

Comment:
Likewise with first comment about babel-runtime? Also, how will this fly with the bower release?